daemon/create.go

```diff
@@ -106,6 +106,10 @@ func (daemon *Daemon) create(params *ContainerCreateConfig) (retC *container.Con
 		return nil, err
 	}
 
+	if err := daemon.updateContainerNetworkSettings(container); err != nil {
```

Comment:
@mavenugo  This method does be called only the first time, because `create` method only be called once. Once the container has been created, the method will never been called again.